pyplanet/apps/contrib/dedimania/__init__.py

```diff
@@ -586,7 +586,10 @@ async def chat_current_record(self):
 		if records_amount > 0:
 			first_record = self.current_records[0]
 			message = '$0b3Current Dedimania Record: $fff\uf017 {}$z$s$0b3 by $fff{}$z$s$0b3 ($l[http://dedimania.net/tm2stats/?do=stat&UId={}&Show=RECORDS]$fff{}$0b3 records$l)'.format(
-				times.format_time(first_record.score), first_record.nickname, self.instance.map_manager.current_map.uid, records_amount
+				times.format_time(first_record.score),
+				(lambda x: x if not (x[-1] == '$' and x[-2:].count('$') == 1) else x[:-1])(first_record.nickname),
```

Comment:
I suspect this is to avoid styling to continue into the rest of the text?
Is there anything specific that the $z$s does not catch?

Reply:
This is correct.
I noticed that there will be an issue if the name of the player ends with a single $.
In this case $$z will be interpreted as $z which leads to not resetting the formating.

For dedimania the file __init__.py is probably the only location to catch this.

Reply:
Ah, I wasn't aware of that! Thanks for your response 👍